lib/tasks/foreman_openscap_tasks.rake

```diff
@@ -56,6 +56,14 @@ namespace :foreman_openscap do
     ForemanOpenscap::MessageCleaner.new.clean
     puts 'Done'
   end
+
+  desc "Delete ArfReports without openscap proxy"
+  task :clean_reports_without_proxy => :environment do
```

Comment:
I suppose we don't want to surprise user which is why we don't delete data automatically through migration, in that case, could you please open a PR against the manual describing how and why to use this?